oktapam/provider_sdkv2.go

```diff
@@ -0,0 +1,170 @@
+package oktapam
```

Comment:
this file seemed to have been renamed from `provider.go`, but the history isn't linking.  can we move it back to `provider.go` so we can preserve history?

Reply:
Yes, it has been renamed from `provider.go`, which uses SDK version 2 with Protocol version 5. We made this change to separate the two providers and to allow `provider_factory.go` to manage their combination. I'm not quite sure what you mean by the "history" of this.